saleor/static/scss/storefront/components/_cart-dropdown.scss

```diff
@@ -1,18 +1,65 @@
 .cart-dropdown {
   position: absolute;
   width: 33rem;
-  top: 0;
-  padding-top: $global-padding * 4;
-  right: $global-padding;
+  top: 6.4rem;
+  right: 0;
   z-index: 3;
   display: none;
+  border: solid 1px $gray;
+  border-radius: 4px 0 4px 4px;
+  background-color: $white;
   .container {
-    background-color: $light-gray;
-    padding-top: $global-padding * 2;
-    padding-bottom: $global-padding * 2;
-    box-shadow: 3px 3px 3px 0px rgba($gray, 0.75);
+    padding-top: $global-padding*2;
+    padding-bottom: $global-padding*2;
     max-height: 28rem;
-    overflow-y: scroll;
+  }
+  &__list {
+    max-height: 10rem;
+    overflow-y: auto;
+    overflow-x: hidden;
+    border-top: solid 1px $gray;
+    border-bottom: solid 1px $gray;
+    &.overflow {
+      overflow-y: hidden;
+    }
+    background: $white;
+    .item {
+      height: 5rem;
+      border-bottom: solid 1px $gray;
+      h3 {
+        font-size: 1rem;
+        margin-bottom: 0;
+        padding: $global-padding*1.4;
+        span {
+          color: $dark-gray;
+        }
+        p {
+          margin-bottom: 0;
+          padding-top: $global-padding*0.2;
+          color: $dark-gray;
+        }
+      }
+      &:last-child {
+        border-bottom: 0;
+      }
+    }
+  }
+  &__total {
+    h3 {
```

Comment:
You can add $font-size-h3: 1rem to variables.scss. So all project will use this style for h3 elements. Also h3 is always uppercased so we can define this style for all document too.